ynr/apps/candidates/tests/test_models.py

```diff
@@ -80,6 +82,27 @@ def test_uncontested_when_winner_count_is_more_than_memberships(self):
         self.create_memberships(self.ballot, parties)
         self.assertFalse(self.ballot.uncontested)
 
+    def test_mark_uncontested_winners(self, log=True):
+        self.factory = RequestFactory()
+        request = self.factory.get("/customer/details")
+        self.user = User.objects.create_user(
+            username="jacob", email="jacob@…", password="top_secret"
+        )
+        request.user = self.user
+
+        parties = self.create_parties(2)
+        self.create_memberships(self.ballot, parties)
+        if not self.ballot.uncontested:
```

Comment:
Instead of this, how about using a standard test assert?

```
self.assertTrue(self.ballot.uncontested)
```
As by having this `return` you might have a test that is silently failing, but still looks like it passes